cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -823,7 +823,7 @@ features:
         title: Configure a Pipeline
         description: Specify the name of the pipeline.
       pipelinenameplaceholder: Pipeline Name
-      headerlabel: Publish Pipeline
+      headerlabel: Deploy Pipeline
```

Comment:
@elfenheart I used deploy because that's the terminology we're using everywhere - on the studio button, on the loading modal, etc. Also, this is not only used for drafts, it is also used for actions that deploy pipelines.